client/src/components/widgets/ContextMenu/template.html

```diff
@@ -0,0 +1,75 @@
+<div>
+  <v-menu v-model="showMenu"
+    :position-x="pos[0]"
+    :position-y="pos[1]"
+    absolute
+    offset-y
+  >
+    <v-card style="padding: 5px">
+      <v-list>
+        <v-list-item dense @click="showRangeDialog=true">
+          <v-list-item-title>
+            Set global range
+          </v-list-item-title>
+        </v-list-item>
+      </v-list>
+      <v-divider/>
+      <v-list>
+        <v-list-item dense @click="fetchImage('png')">
+          <v-list-item-title>
+            Save Image as PNG
+          </v-list-item-title>
+        </v-list-item>
+        <v-list-item dense @click="fetchImage('pdf')">
+          <v-list-item-title>
+            Save Image as PDF
+          </v-list-item-title>
+        </v-list-item>
+      </v-list>
+      <v-divider/>
+      <v-list>
+        <v-list-item dense @click="fetchMovie('mp4')">
+          <v-list-item-title>
+            Download Movie as MP4
+          </v-list-item-title>
+        </v-list-item>
+        <v-list-item dense @click="fetchMovie('mpg')">
+          <v-list-item-title>
+            Download Movie as MPG
+          </v-list-item-title>
+        </v-list-item>
+      </v-list>
+    </v-card>
+  </v-menu>
+  <range-dialog
+    :visible="showRangeDialog"
+    :param="parameter"
+    @close="showRangeDialog=false"
+  />
+  <div class="notification">
+    <v-sheet
+      v-for="r in requested"
+      :key="r.id"
+      :elevation="12"
+      color="blue"
```

Comment:
Can we use the MD "success", "error" colors from the theme? Lets also use the `v-snackbar`. We should also render the latex nicely as well.